Source/Core/DolphinNoGUI/MainNoGUI.cpp

```diff
@@ -260,6 +268,17 @@ int main(int argc, char* argv[])
 
   const WindowSystemInfo wsi = s_platform->GetWindowSystemInfo();
 
+  // WindowSystemInfo prepared_wsi(wsi);
+
+  // g_video_backend->PrepareWindow(prepared_wsi);
+
+  // VideoBackendBase::PopulateBackendInfo();
+  // if (!g_video_backend->Initialize(prepared_wsi))
+  // {
+  //   PanicAlertFmt("Failed to initialize video backend!");
+  //   return -1;
+  // }
+
```

Comment:
This commented out code shouldn't be in this PR

(and it's not worth keeping around as a comment, AsyncPresent will change how initialisation is done)

Reply:
Yeah that's my fault I was messing with stuff, and that's what prompted this pr